docs/pages/reference/agent-services/database-access-reference/cli.mdx

```diff
@@ -399,10 +399,10 @@ $ tsh db exec "select 1" --db-user mysql --labels env=dev --parallel=5 --output-
 | `--db-user` | The database user to log in as. |
 | `--db-name` | The database name to log in to. |
 | `--db-roles` | List of comma separate database roles to use for auto-provisioned user. |
-| `--dbs` |  List of comma separated target database services. Mutually exclusive with `--search` or `--labels`. |
+| `--dbs` |  List of comma separated target databases. Mutually exclusive with `--search` or `--labels`. |
```

Comment:
"database services" is correct in this instance I think, otherwise the difference between `--dbs` and `-db-name` might not be clear enough.